README.md

```diff
@@ -1,3 +1,163 @@
-# Google Drive App
+# Mattermost/Google Drive Integration
 
-A Google Drive app for Mattermost.
+* [Feature summary](#feature-summary)
+* [Set up](#set-up)
+    * [Installation HTTP](#installation-http)
+    * [Installation Mattermost Cloud](#installation-mattermost-cloud)
+    * [Create a Google Cloud Project](#create-a-google-cloud-project)
+    * [Configuration](#configuration)
+* [Admin guide](#admin-guide)
+    * [Slash commands](#slash-commands)
+* [End user guide](#end-user-guide)
+    * [Get started](#get-started)
+    * [Use /drive commands](#use-drive-commands)
+    * [Post menu bindings](#post-menu-bindings)
+* [Development](#development-environment)
+    * [Manual installation](#manual-installation)
+    * [Install dependencies](#install-dependencies)
+    * [Run the local development environment](#run-the-local-development-environment)
+    * [Run the local development environment with Docker](#run-the-local-development-environment-with-docker)
+
+# Feature summary
+
+This application allows you to integrate Google Drive to your Mattermost instance, letting you:
+
+- Create a Google Drive file
+- Share a Google Drive file
+- View and reply to comments
+- Publish on Google Drive any file attached to a Mattermost post
+- Enable or disable notifications for all files
+
+# Set up
+
+## Installation HTTP
+
+To install, as a Mattermost system admin user, run the command ``/apps install http GOOGLE_API_URL`` in any channel. The ``/drive`` command should be available after the configuration has been successfully installed.
+
+The ``GOOGLE_API_URL`` should be replaced with the URL where the Google Drive API instance is running. Example: ``/apps install http https://myapp.com/manifest.json``
+
+## Installation Mattermost Cloud
+
+To install, as a Mattermost system admin user, run the command ``/apps install listed drive`` in any channel. The ``/drive`` command should be available after the configuration has been successfully installed.
+
+## Create a Google Cloud Project
+
+1. Create a new Project. You would need to redirect to [Google Cloud Console](https://console.cloud.google.com/home/dashboard) and select the option to **New project**. Then, select the name and the organization (optional).
+2. Select APIs. After creating a project, on the left side menu on **APIs & Services**, then, select the first option **Enabled APIs & Services** and wait, the page will redirect. 
+3. From the left menu select **Library** and activate: 
+    - Google Drive API
+    - Google Docs API
+    - Google Slides API
+    - Google Sheets API
```

Comment:
It also expects Google Drive Activity API to be enabled. If not, I get the following error if notifications are enabled:

```
ERROR: Drive Activity API has not been used in project <PROJECT_ID> before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/driveactivity.googleapis.com/overview?project={PROJECT_ID} then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
```

Reply:
Thanks! This is resolved in commit [6d9ef5d](https://github.com/mattermost/mattermost-app-google-drive/pull/2/commits/6d9ef5dd391b63393237c569a1efad004dd19c7d)